tmexp/__main__.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import logging
+from typing import Any
+
+from .preprocess import preprocess
```

Comment:
> `.preprocess` > `preprocess`

Reply:
it's not a typo, it's a relative import (new in python3). On another note, I solved the other small problems, should be good to merge :)